scripts/kani-regression.sh

```diff
@@ -55,15 +55,10 @@ for testp in "${TESTS[@]}"; do
   testl=($testp)
   suite=${testl[0]}
   mode=${testl[1]}
-  echo "Check compiletest suite=$suite mode=$mode ($TARGET -> $TARGET)"
```

Comment:
I think we are not using `$TARGET` anywhere else so the platform test above can be removed.